sync/src/relayer/tests/compact_block_process.rs

```diff
@@ -144,50 +143,6 @@ fn test_unknow_parent() {
     );
 }
 
-#[test]
```

Comment:
I think this test case is still helpful.
So if we change to accept the not-better block, we should change the test case logic, but not remove it. Or add other tests instead.

Reply:
updated.